### DISCUSSION TITLE
Album covers (LastFM API and others)

### DISCUSSION THREAD

**Discussion by ivi-v** [AzuraCast/AzuraCast#7522] [Q&A]
Hello!

I just found the album cover service and I've using it recently with both "web services" options checked. The thing is that it only searches on the "coverartarchive" service, which was down for some weeks due to DDoS attacks, and it only shows some covers and not everyone is the correct one.

I tried to use a LastFM API key as it is indicated, but it doesn't work. Maybe do I need to configure something on the own LastFM service? And in addition, do you know any other service supported by AzuraCast in order to show the correct album covers on the AzuraCast player?

Thank you in advance!

**Comment by ivi-v** [comment]
🙏🙏🙏 Thank you so much

**Reply by gAlleb** [reply]
Hi! I'm not aware of any other cover art services supported by AC. 

`Last.fm` is a disaster in terms of cover art.

Other services like iTunes, Spotify are not supported due to some "legal" rules and requirements they impose.

Best cover art service - is tagging all your files with relevant cover art.

Alternatively you can make/write a custom page/site for your AzuraCast instance and attach any cover art service to it. Or use some free solutions like

https://github.com/PeWe79/AzuraCast-Player

![image](https://github.com/user-attachments/assets/5c5e9456-53dc-4e9f-a504-531a03fd7c0b)
